backend-rust/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.1.52] - 2025-05-06
```

Comment:
When you bump the version to `0.1.52`, then you need to update the version in `Cargo.toml` file accordingly (see `Readme` in top level of the repo) and make a release. Alternatively, you can keep this under `[unrelease]` section and not increment the version.

Reply:
I have added unreleased section in the changelog, please review